releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```
@@ -0,0 +1,70 @@
+---
+prelude: |
+  We are very pleased to release Qiskit 1.0.0. This release is the
+  culmination of 7 years of development to mature Qiskit into a stable high
+  performance SDK to perform quantum computation, and is the start of new
+  era for the Qiskit project and community. The most significant changes with the 1.0.0 
+ release relate to the stability policy, release cycles, and versioning.
+  Qiskit is now formally using `semantic versioning <https://semver.org/>`__ ,
+  and this means that, for the entire lifecycle of the 1.x release series, the project
+  is commited to maintaining backwards compatibility. We're also now starting
+  to offer bugfix support for major versions, so that you continue to have a
+  supported branch for 6 months after the release of the next major version.
+  This starts now with the 1.x and 0.x major series: the 0.46.x release will
+  continue to be supported, with bugfixes and periodic patch releases, for 6
```

```suggestion
  continue to be supported with periodic bugfixes **only**, for 6
```

---

"periodic patch releases" sounds (to me) like we would supply new features, too...

---

Yeah I can see that. The patch release terminology comes directly from semver but if you're not familiar with that I could see it being confusing. How about something like?:

```suggestion
  continue to be supported, with periodic patch releases that contain bug fixes, for 6
```

---

Sounds good, thanks! :-)